release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2158,6 +2158,33 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-6-known-issues"]
 == Known issues
 
+* The GPU Operator and Node Feature Discovery (NFD) Operator are not available on a fresh installation of {product-title} 4.6.1. To use NFD in version 4.6.1, you must install {product-title} 4.5 and upgrade the cluster to version 4.6.1. 
++
+Another workaround on a fresh installation of {product-title} 4.6.1 is to install the NFD Operator from its associated GitHub repository using the CLI.
++
+. Clone the NFD Operator GitHub repository:
++
+[source,terminal]
+----
+$ git clone --branch release-4.5 https://github.com/openshift/cluster-nfd-operator.git
+----
++
+. Navigate into the new NFD Operator repository folder:
++
+[source,terminal]
+----
+$ cd cluster-nfd-operator
+----
++
+. Set the pull policy for the repository to always deploy the NFD Operator, so it is deployed to your running cluster:
++
+[source,terminal]
+----
+$ PULLPOLICY=Always make deploy
```

Comment:
You need to quote this:

```console
$ PULLPOLICY=Always make deploy
make: *** No rule to make target `deploy'.  Stop.
```

Reply:
oops, you are expecting to call `make`.  Sorry for the noise

Reply:
@wking like this?
```
$ PULLPOLICY="Always make deploy"
```

Reply:
Your second comment just rendered for me :thinking: 

Is the original one good then?
```
$ PULLPOLICY=Always make deploy
```

Reply:
@codyhoag your updated version LGTM.  thank you